src/rt/test/systematic.h

```diff
@@ -443,8 +443,8 @@ namespace Systematic
   inline static void signal_handler(int sig, siginfo_t*, void*)
   {
     auto str = strsignal(sig);
-    write(1, str, strlen(str));
-    write(1, "\n", 1);
+    (void)write(1, str, strlen(str));
```

Comment:
Can we use the UNUSED macro for clarity?

Reply:
So far we don't use UNUSED for complex expressions, only for function parameters.

I'd be happy to switch it, as it expands to the same thing, but I don't really like putting things with side effects into macro calls.

Reply:
Should I just add a comment above instead.